src/components/menus/Drawer/Drawer.tsx

```diff
@@ -1,19 +1,26 @@
 import * as React from 'react';
 import classnames from 'classnames';
-import * as styles from './Drawer.sass';
+import * as styles from './Drawer.scss';
 import IReactComponentProps from '../../../common/structures/IReactComponentProps';
 
 interface IProps extends IReactComponentProps {
 	align?: 'left' | 'center' | 'right';
 	children: React.ReactNode;
 	show?: boolean;
+	stripes?: boolean;
+	shadow?: boolean;
 }
 
 interface IState {
 	disableAnimation: boolean;
 }
 
 export default class Drawer extends React.Component<IProps, IState> {
+	static defaultProps: Partial<IProps> = {
+		stripes: true,
```

Comment:
Can we make the default `stripes: false`? Since the designs so far don't include stripes & that way the implementation of this component could be `<Drawer stripes />` instead of `<Drawer stripes={false} />` ?

Reply:
I'm hesitant because defaulting to true is just defaulting to existing behavior, which I feel like is safest. I could actually change it to `noStripes`, that would preserve existing any existing implementation without refactoring and allow for the implicit prop passing - neat!